app.rb

```diff
@@ -1,7 +1,21 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/opponent'
+require './lib/game'
+
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+  get '/' do
+    erb :index
+  end
+
+  post '/result' do
+    # should this be lowercase or capitalize?
+    # if i dont capitalize, something breaks, reviewer pls
+    @name = params[:name].capitalize
+    @player_hand = params[:choice]
+    @computer_hand = Opponent.new.computer_hand
+    @winner = Game.new.winner(@player_hand.downcase.to_sym, @computer_hand.downcase.to_sym)
```

Comment:
Is instantiating a new Opponent Object and Game Object every time you play one round the best idea?

Reply:
nah, couldve used a single instance here for sure. will consider this for future